### DISCUSSION TITLE
[Feature] Auto-retry checkbox for uploading

### DISCUSSION THREAD

**Discussion by F1LIP3** [immich-app/immich#6217] [Feature Request]
### The feature

Having a ability to choose or not if you want the web or mobile apps to auto-retry the files that were skipped due to error when uploading.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by adilshoukat** [comment]
I think having a retry button will be much convenient. In case the wifi got disconnected or you mistakenly moved out from the wifi range, you should be able to just hit **retry / retry all** rather than cherry picking the files with the same name from the gallery again.

**Comment by TheMClaeys** [comment]
This would be most helpfull.
I'm importing some picture folders from my computer where I didn't know if they were already in Immich and it takes a lot of scrolling to find these that failed:
<img width="312" height="99" alt="image" src="https://github.com/user-attachments/assets/8cb642c4-820e-4c74-bbed-53b5d71eaec3" />

Or if you click the failed number that it only shows the failed. That's less helpfull for someone with a lot of failed uploads though as they have to click them one by one.

**Comment by claude-on-call** [comment]
Another concrete data point for this: a batch of mobile background-backup uploads had 37 files fail with a mix of `413`, `502`, `520`, and `524` errors. Most of those (502/520/524) look like exactly the transient proxy/server hiccups this thread is about — but there was no way to re-queue any of them short of hoping a future automatic backup pass happens to pick them back up. No per-item retry, no bulk retry, on mobile.

+1 for adilshoukat's framing above — "retry / retry all" covers it well. Even just a manual per-item retry on the failed-uploads list would already be a big improvement over the current "hope it eventually retries itself" situation.
